lib/output/output.go

```diff
@@ -1,11 +1,8 @@
 package output
 
-import (
-	"encoding/json"
-	"fmt"
-	"sort"
-	"strings"
-)
+type OutputBuilder interface {
```

Comment:
I think 'struct typing' allows any struct that minimally implements the interface, but may implement methods outside the interface, to be used as the interface. For example: https://play.golang.org/p/2JS1qC9kca